src/Support/Str.php

```diff
@@ -85,6 +85,10 @@ public static function stripTags($html, $tags_list = [])
 
     public static function slug($string, $separator = '-', $language = 'en')
     {
+        if ($string === null) {
```

Comment:
can we set `$string = ''` instead for less code?

Reply:
Explicitly passing `null` for `$string` would still lead to the log entry, unfortunately.

Reply:
that's not what I meant. I meant this `public static function slug($string = ''...` @JohnathonKoster

Reply:
Right, but if a value is supplied that is `null`, it wouldn't automatically get replaced with that empty string (only if omitted entirely), and still fall through and trigger the warning on the first `str_replace`.

Reply:
Wouldn't really want to set a default for the first argument anyway since passing nothing to it doesn't really make any sense.